fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1069,6 +1069,53 @@ def assert_state(self, state: TrainingState) -> None:
             self.training_state == state
         ), f"expected to be in state {state} but current state is {self.training_state}"
 
+    @staticmethod
+    @contextlib.contextmanager
+    def enable_auto_wrap(**kwargs) -> Iterator[None]:
```

Comment:
Also, why return Iterator[None] type not Generator type? A comment might be nice.